autoscheduler/frontend/src/utils/sortMeetingFunction.tsx

```diff
@@ -0,0 +1,8 @@
+import Meeting from '../types/Meeting';
+
+// sorts meeetings based on meeting type
+const sortMeeting = (a: Meeting, b: Meeting): number => {
+  const value = a.meetingType - b.meetingType;
+  return value;
+};
```

Comment:
```suggestion
const sortMeeting = (a: Meeting, b: Meeting): number => a.meetingType - b.meetingType;
```
Since this immediately returns a value, we can use this shorthand instead